src/VirtoCommerce.Platform.Web/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -107,5 +110,30 @@ public static IServiceCollection AddSecurityServices(this IServiceCollection ser
 
             return services;
         }
+
+
+
+        public static IServiceCollection AddLibraries(this IServiceCollection services)
```

Comment:
It seems will be good to rename to AddUnmanagedLibraries or AddNativeLibraries .